tests/integration/ha_tests/test_ha.py

```diff
@@ -657,7 +657,7 @@ async def test_network_cut(ops_test, continuous_writes):
 
 @pytest.mark.abort_on_fail
 @pytest.mark.unstable
-async def test_scale_up_down(ops_test: OpsTest, continuous_writes):
+async def test_scale_up_down(ops_test: OpsTest):
```

Comment:
why remove `continous_writes`? I am fairly certain this must be here

some context: `continous_writes` is a fixture which applies a continous stream of writes to the DB, these writes and then used to verify if replication is operating correctly, or if the database properly recieves writes in edge case scenarios (ie killing a primary, scaling, up and scaling down). 

AFAICT removing this means that this test is no longer testing is the database can replicate writes during scaling.

Reply:
We must check continuous writes in HA tests.

Reply:
restored